projects/storefrontlib/shared/components/media/media.service.ts

```diff
@@ -38,7 +38,7 @@ export class MediaService {
    * for specific formats.
    */
   getMedia(
-    mediaContainer?: MediaContainer | Image,
+    mediaContainer: MediaContainer | Image,
```

Comment:
Why change this file? This doesn't seem to fit the scope of the PR.
Let's discuss.

Reply:
Don't know why this is here. I didn't make any changes to this file. Maybe during a /develop branch merge it changed these files